laika/__init__.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from .astro_dog import AstroDog
-assert AstroDog
+assert AstroDog # type: ignore
```

Comment:
why is this needed now?

Reply:
mypy will throw https://mypy.readthedocs.io/en/latest/error_code_list.html?highlight=truthy-function#check-that-function-isn-t-used-in-boolean-context-truthy-function since normally the name of an existing class (AstroDog) will always evaluate to True.

I didn't wanted to remove the test

Reply:
we need to be very careful with ignores. this one seems to silently turn off a bunch of coverage in the `astro_dog.py` file